packages/cmd/src/CommandPreparer.ts

```diff
@@ -28,9 +29,10 @@ export class CommandPreparer {
      * Prepare the command definition and apply any pass on traits to children.
      * After a definition has been prepared, it should be considered final.
      * @param {ICommandDefinition} original - The original command definition tree to "prepare"
+     * @param {ICommandProfileTypeConfiguration} baseProfile - An optional base profile to add to command definitions
      * @return {ICommandDefinition} A copy of the original that has been prepared
      */
-    public static prepare(original: ICommandDefinition): ICommandDefinition {
+    public static prepare(original: ICommandDefinition, baseProfile?: ICommandProfileTypeConfiguration): ICommandDefinition {
```

Comment:
I believe this was one of the implementation decisions made in order to make it backwards compatible.
Looks good though! 👍